src/Faker/Provider/hi_IN/Text.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace Faker\Provider\hi_IN;
+
+class Text extends \Faker\Provider\Text
+{
+    protected static $textStartsWithUppercase = false;
+
+    protected static function appendEnd($text)
+    {
+        return rtrim($text, ',— ').'।';
+    }
+
+    /**
+     *
+     *
+     * Title: हमारी अपनी सुरक्षा प्रणाली और
+     *        उसके जुझारू सैनिक
+     *
+     * Author: डॉ गुरुदयाल प्रदीप
+     *
+     *
+     * Language: Hindi
+     *
+     * @link http://www.abhivyakti-hindi.org/vigyan_varta/vigyan/2007/suraksha_pranali.htm
```

Comment:
please use a more generalist piece of work, preferably a famous XXth Century novel in the public domain.